apps/react-workshop/src/DatePicker.stories.tsx

```diff
@@ -58,21 +58,24 @@ export const WithTime = () => {
 
   return (
     <>
-      <IconButton onClick={() => setOpened(!opened)} id='picker-button'>
-        <SvgCalendar />
-      </IconButton>
+      <Popover
+        content={
+          <div style={{ marginTop: 4 }}>
```

Comment:
Can you also remove the `marginTop`? You can use [`middleware.offset`](https://itwinui.bentley.com/docs/popover#positioning) if you want to keep the gap.

Reply:
Although, I wonder if we even need the gap. 🤔

Opinions? @rohan-kadkol

Reply:
I'll remove the `<div>` and use `middlewar.offeset` to maintain the gap for now.

Reply:
> Although, I wonder if we even need the gap. 🤔

I'm not sure of the reason, but looking at the git history, looks like this was added back in April 2021 ([iTwin/iTwinUI-react #4 (files)](https://redirect.github.com/iTwin/iTwinUI-react/pull/4/files#diff-d92e6fc01405e3a187caa82f9fb11ed2bc03ba56c71ff84750c3b22885e16709L48-R48)) and so may be outdated now with the new `Popover` component.

Reply:
Yeah lets get rid of the gap then. We don't show a gap in the website demos https://itwinui.bentley.com/docs/datepicker

Reply:
`div` removed and `offset` added in this [commit](https://github.com/iTwin/iTwinUI/pull/2077/commits/f66d1b6a478a9a471995cecd614c3b5f6d0fed5d).

Reply:
Sorry, github didn't show me the follow up comments until now. Removed `offset` gap and updated test images to match in this [commit](https://github.com/iTwin/iTwinUI/pull/2077/commits/c9357d0f11a014cb0a7d990197e7960166fe8b05).